modules/widgets/eu-cookie-law/footer.php

```diff
@@ -0,0 +1,77 @@
+<div
+	class="<?php echo 'negative' === $instance['color-scheme'] ? 'negative ' : '';
+	?>hide-on-<?php echo esc_attr( $instance['hide'] ); ?>"
+	data-hide-timeout="<?php echo intval( $instance['hide-timeout'] ); ?>"
+	id="eu-cookie-law"
+>
+	<form method="post">
+		<?php wp_nonce_field( 'eucookielaw' ); ?>
+		<input type="hidden" name="eucookielaw" value="accept" />
+		<input type="hidden" name="redirect_url" value="<?php echo esc_attr( $_SERVER['REQUEST_URI'] ); ?>" />
+		<input type="submit" value="<?php echo esc_attr( $instance['button'] ); ?>" class="accept" />
+	</form>
+
+	<?php if ( 'default' == $instance['text'] || empty( $instance['customtext'] ) ) {
+		echo $defaults['default-text'];
+		?>
+		<br />
+		<?php
+		_e( 'To find out more, as well as how to remove or block these, see here:', 'jetpack' );
+		echo ' ';
+	} else {
+		echo esc_html( $instance['customtext'] ), ' ';
+	} ?>
+
+	<a href="<?php echo esc_attr(
+		'default' === $instance['policy-url'] || empty( $instance['custom-policy-url'] )
+			? $defaults['default-policy-url']
+			: $instance['custom-policy-url']
+	); ?>" >
+		<?php echo esc_html( $instance['policy-link-text'] ); ?>
+	</a>
+</div>
+
+<script type="text/javascript">
+	jQuery( function( $ ) {
```

Comment:
We rely on jQuery here, but we didn't make sure it was actually enqueued on the page before.